pkg/platform/runtime2/store/store.go

```diff
@@ -253,13 +253,6 @@ func (s *Store) updateEnviron(orderedArtifacts []artifact.ArtifactID, artifacts
 		}
 	}
 
-	cnst := envdef.NewConstants(s.InstallPath())
-	rtGlobal = rtGlobal.ExpandVariables(cnst)
-	err := rtGlobal.ApplyFileTransforms(s.InstallPath(), cnst)
-	if err != nil {
-		return nil, locale.WrapError(err, "runtime_alternative_file_transforms_err", "", "Could not apply necessary file transformations after unpacking")
-	}
-
```

Comment:
I moved this block into the `setupArtifact` function for these reasons:

- makes this function more unixy. :)
- the relocation can and should be parallelized (can be applied for each artifact individually)
- in the `setupArtifact` function the relocation is more isolated, as it is run inside of the temporary installation directory where only one artifact is unpacked to.